### ISSUE TITLE
add option to add price alerts

### ISSUE THREAD

**Issue by alex-tee** [bitstower/markets#39]
hi, thanks for this!

is it possible to add an option to specify price alerts that would show in the desktop environment (like gnome alerts)? also interested in using this on the pinephone, i guess phone distros will have a similar alert/notification system

**Comment by tomasz-oponowicz** [comment]
hey @alex-tee,

It makes total sense to me. Though we need the detailed view #26 first in order to squeeze all those extra features.

**Comment by alex-tee** [comment]
:+1: btw are there plans to make this a full blown app with technical analysis tools (like candle graphs and moving averages)? something like tradingview. I'm looking for a complete replacement to proprietary tools but I can't seem to find anything

**Comment by tomasz-oponowicz** [comment]
Not really. The idea is to have a simple market tracker for PinePhone. The detailed view #26 (see the scope in the ticket) is the last missing piece.
